rpms/SPECS/wazuh-manager.spec

```diff
@@ -338,10 +340,6 @@ if [ $1 = 1 ]; then
   %{_localstatedir}/packages_files/manager_installation_scripts/add_localfiles.sh %{_localstatedir} >> %{_localstatedir}/etc/ossec.conf
 fi
 
```

Comment:
Why we are removing this:
```

if [[ -d /run/systemd/system ]]; then
  rm -f %{_initrddir}/wazuh-manager
fi
```

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/2299/commits/c78f49f962d59bee37ef0c2f165b150cb4371f4b well spotted @lchico, thanks!